src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -1598,6 +1604,82 @@ static void G()
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void IdentityConversion_LocalDeclaration()
```

Comment:
I don't know if it's covered elsewhere, but consider testing an `IMixed<in TIn, out TOut>` too